src/types/search.d.ts

```diff
@@ -40,6 +40,12 @@ declare class Search {
     userParameters?: UserParameters,
     networkParameters?: NetworkParameters
   ): Promise<SearchResponse>;
+
+  getVoiceSearchResults(
+    query: string,
+    parameters?: Omit<SearchParameters, 'filters' | 'sortBy' | 'sortOrder'>,
+    networkParameters?: NetworkParameters
```

Comment:
I think we missed out userParameters here

Reply:
Nice catch!